### PR TITLE
added a new method for tag_to_span

### PR DESCRIPTION
**Pull request by omri374**
Helper method to perform tag labeling to span labeling (opposite of `span_to_tag` for cases where models return tags but we want to calculate span based metrics.

### CODE REVIEW

File: presidio_evaluator/tag_to_span.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+def tag_to_span(tokens: Doc, tags: List[str]) -> List[Span]:
+    """
+    Transforms a list of tags to a list of spans, while ignoring labeling scheme.
+    :param tokens: List of spaCy tokens
+    :param tags: List of tags (e.g. ["O", "B-PER", "I-PER", "U-LOC"])
+    :return: List of spans
+    """
+
+    # If IO, translate to BIO for easier manipulation
```
Comment (tranguyen221) [inline presidio_evaluator/tag_to_span.py:21]: should it be "if it is not IO, translate to IO for easier manipulation"?

File: tests/test_tag_to_span.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            [[2, 3, "name"], [3, 4, "loc"]],
+        ),
+        (
+            "Dan is Washington",
+            ["U-name", "O", "U-loc"],
+            [[0, 1, "name"], [2, 3, "loc"]],
+        ),
+    ],
+)
+def test_tag_to_span(spacy_en_core_web_sm, text, tags, ents):
```
Comment (tranguyen221) [inline tests/test_tag_to_span.py:34]: It's great! The function works well. I also add the other test cases and all of them passed. Thanks @omri374 for working on it.

**Review by tranguyen221 — COMMENTED** [review]
Thanks @omri374. I also tested the function with the other cases and it works well!

File: presidio_evaluator/tag_to_span.py (hunk 1 above)
Comment (omri374) [reply presidio_evaluator/tag_to_span.py:21]: Yes! good catch

File: tests/test_tag_to_span.py (hunk 2 above)
Comment (omri374) [reply tests/test_tag_to_span.py:34]: Thanks!

**Review by tranguyen221 — APPROVED** [review]

**Pull request merged**